submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,29 @@
+
+### Nauvis is a white dwarf
+
+This is a short recap of the Reddit post by [u/DaveMcW](https://www.reddit.com/user/DaveMcW/) that I just mentioned. We know the length of one day/night cycle on the planet, which gives us its angular velocity. By stating that the centripetal force from the planet's rotation has to be less than gravity at the equator and then cancelling a bunch of values, we get that Nauvis's density has to be greater than 813400 kg/m\^3. That makes Nauvis at least as dense as a white dwarf star.
+
+### A bit about Nauvis's orbital mechanics
+
+Since the shadows are at a constant length and direction throughout the day, the only possible explanation is that the light source doesn't move, meaning Nauvis is tidally locked. This means the day/night cycle is caused by a different object occluding the light. Based on the frequency of the nights, that object is probably in orbit around Nauvis, and it can't be a moon, since a moon large enough for such long nights would tidally lock Nauvis to itself. So night must be caused by a cloud of debris in orbit of Nauvis, which also gives a reason for the Engineer crashing there in the first place.
+
+### Nauvis reflects at least 88.3% of all light that hits it
+
+A single solar panel has a 9 m\^2 area and produces 60 kW during daytime. From that number we find that the surface receives at least 6.67 kW/m\^2 (compare to 1 kW/m\^2 for Earth). Considering Nauvis's ambient temperature is 15 degrees Celsius, we can use the [effective temperature formula](https://en.wikipedia.org/wiki/Effective_temperature#Surface_temperature_of_a_planet) to find that if Nauvis was a perfect blackbody, it has to have an albedo of 0.883 (keeping in mind that it's tidally locked). For comparison, the Earth has an albedo of 0.306, the Moon has an albedo of 0.12, and Enceladus, the most reflective body in the Solar system, has an albedo of 0.9. And all of that without even considering that solar panels have less than 100% efficiency, and that Nauvis is not an ideal blackbody!
+
+### Nuclear fuel is 3 times hotter than the surface of the Sun
+
+Now it's time to come back from space and take a look at the artificial things. Locomotives have a constant energy consumption of 600 kW, but better fuel makes them faster. This means higher-tier fuel increases the engine's efficiency. Since useful power is force times velocity, efficiency is proportional to acceleration times top speed. Now, since we can use any fuel to power the locomotive, we can safely assume it is based on a [heat engine](https://en.wikipedia.org/wiki/Heat_engine). If we assume that the engine is a perfect heat engine that works using the [Carnot cycle](https://en.wikipedia.org/wiki/Carnot_cycle), and that coal burns at a temperature of 165 degrees Celsius (based on boiler steam temperature) and ambient temperature is 15 degrees, after plugging in all the numbers we get that nuclear fuel has a temperature of over 18000 degrees.
```

Comment:
Maybe add a screenshot below this of your maths laid out nicely (maybe using an online equation writing tool) then focus a bit more on what this means in the text rather than just stating the maths. Could also use some perspective on the final output, how does this compare to other stars and objects?